specs/agents/tracing-instrumentation-http.md

```diff
@@ -35,6 +35,30 @@ Request and response headers, cookies, and form bodies should be sanitised (i.e.
 
 Agents may may include additional patterns if there are common conventions specific to language frameworks.
 
+##### `transaction_ignore_urls` configuration
+
+Used to restrict requests to certain URLs from being instrumented.
+
+This property should be set to a list containing one or more strings.
+When an incoming HTTP request is detected,
+its request path will be tested against each element in this list.
+For example, adding `/home/index` to this list would match and remove instrumentation from the following URLs:
+
+```
+https://www.mycoolsite.com/home/index
+http://localhost/home/index
+http://whatever.com/home/index?value1=123
+```
```

Comment:
This does not work like the `WildcardMatcher` in the provided [examples](https://github.com/elastic/apm/blob/d32077ad5bcc847ddc9b4f9d3f13f04624a6decc/tests/agents/json-specs/wildcard_matcher_tests.json#L85-L92).

This proposal has implicit `*`s around `/home/index` whereas the _original_ `WildcardMatcher` expects explicit `*`s.

Reply:
I don't think so. The query string should not be taken into account for matching, so the path for all three examples is essentially `/home/index`, which matches the `/home/index` pattern without the need for wildcards.

Reply:
Oh, you're right. I was thinking about the whole URL with hostname. Perhaps it would be helpful to provide an example with an effective asterisk?

Reply:
I'll see if I can make it more clear that _only_ the request path should be used for matching